staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode.go

```diff
@@ -58,12 +58,11 @@ var decode cbor.DecMode = func() cbor.DecMode {
 		// with or without tagging. If a tag number is present, it must be valid.
 		TimeTag: cbor.DecTagOptional,
 
-		// Observed depth up to 16 in fuzzed batch/v1 CronJobList. JSON implementation limit
-		// is 10000.
-		MaxNestedLevels: 64,
-
-		MaxArrayElements: 1024,
-		MaxMapPairs:      1024,
+		// MaxNestedLevels, MaxArrayElements and MaxMapPairs are copied from the unexported
+		// max values allowed by fxamacker/cbor package.
+		MaxNestedLevels:  65535,
+		MaxArrayElements: 2147483647,
+		MaxMapPairs:      2147483647,
```

Comment:
Is there an example of a real API object that would contain a map with more than a thousand key/value pairs?